pkg/metrics/metrics.jsx

```diff
@@ -1632,10 +1632,17 @@ class MetricsHistory extends React.Component {
                 .catch(ex => this.setState({ error: ex.toString() }));
     }
 
-    componentDidMount() {
-        packagekit.detect().then(exists => {
-            this.setState({ packagekitExists: exists });
-        });
+    async componentDidMount() {
+        const packagekitExists = await packagekit.detect();
+        // HACK: See https://github.com/cockpit-project/cockpit/issues/19143
+        let cmdline = "";
+        try {
+            cmdline = await cockpit.file("/proc/self/cmdline").read();
+        } catch (_ex) {}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19184-20230808-080203-8b12e916-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)